test/admin.test.js

```diff
@@ -285,4 +284,79 @@ describe('AdminApi', function () {
       })
     })
   })
+
+  describe('postBalance', function () {
+    it('adds/subtracts the balance by the given amount', async function () {
+      await this.adminApi.postBalance('', { accountId: 'test.cad-ledger', amountDiff: '12' })
+      await this.adminApi.postBalance('', { accountId: 'test.cad-ledger', amountDiff: '-34' })
+      const balanceMiddleware = this.middlewareManager.getMiddleware('balance')
+      assert.equal(
+        balanceMiddleware.getStatus().accounts['test.cad-ledger'].balance,
+        (12 - 34).toString())
+    })
+
+    it('rejects on invalid BalanceUpdate', async function () {
+      try {
+        await this.adminApi.postBalance('', {})
+      } catch (err) {
+        return
+      }
+      assert(false)
+    })
+  })
+
+  describe('getAlerts', function () {
+    it('returns no alerts by default', async function () {
+      assert.deepEqual(await this.adminApi.getAlerts(), {alerts: []})
+    })
+
+    it('returns an alert when a peer returns "maximum balance exceeded"', async function () {
```

Comment:
can we also add a test to make sure it correctly groups alerts which have been triggered by the same connector?